docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -80,7 +93,10 @@ If you add a new custom policy, it must be defined under a new package like `use
         - A `string` denoting the detected issue
             - Although `object` with `msg` field is accepted, other fields are dropped and `string` is recommended if `result.new()` is not utilised.
             - e.g. `{"msg": "deny message", "details": "something"}`
-    
+
+### Subtypes in the custom data
+
+TODO
```

Comment:
@simar7 what are subtypes?

Reply:
Subtypes are documented [here](https://aquasecurity.github.io/trivy/v0.49/docs/scanner/misconfiguration/custom/selectors/#enabling-selectors-and-subtypes) and are only needed at this time for cloud checks.

They help us only run the checks that are needed for what's being scanned, optimizing the cloud scanning runtime a little.